resources/extruders/geeetech_A30M_extruder0.def.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "geeetech_A30M", 
+        "position": "0" 
+                      },
```

Comment:
This file has very strange formatting. Could you please pass it through a JSON linter?